packages/paste-core/components/separator/src/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Space} from '@twilio-paste/style-props';
+
+export interface SeparatorProps extends React.HTMLAttributes<HTMLHRElement> {
+  id?: never;
+  className?: never;
+  orientation: 'horizontal' | 'vertical';
+  horizontalSpacing?: Space;
+  verticalSpacing?: Space;
+}
+
+const Separator: React.FC<SeparatorProps> = ({orientation, horizontalSpacing, verticalSpacing, ...props}) => {
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      aria-orientation={orientation}
+      margin="space0"
+      marginBottom={verticalSpacing}
+      marginLeft={horizontalSpacing}
+      marginRight={horizontalSpacing}
+      marginTop={verticalSpacing}
+      as="hr"
+      borderWidth="borderWidth0"
+      borderColor={'colorBorderLight'}
+      borderStyle="solid"
+      borderBottomWidth={orientation === 'horizontal' ? 'borderWidth10' : null}
+      borderLeftWidth={orientation === 'vertical' ? 'borderWidth20' : null}
+      width={orientation === 'horizontal' ? 'auto' : null}
+      height={orientation === 'vertical' ? 'auto' : null}
+    />
+  );
+};
+
+Separator.displayName = 'Separator';
+export {Separator};
```

Comment:
We should have `propTypes` for `orientation`, `horizontalSpacing`, and `verticalSpacing`, right?

Reply:
Good catch!